plugins/processors/aws/ec2/ec2.go

```diff
@@ -91,6 +122,12 @@ func (r *AwsEc2Processor) Init() error {
 }
 
 func (r *AwsEc2Processor) Start(acc telegraf.Accumulator) error {
+	r.tagCache = freecache.NewCache(r.TagCacheSize)
+	if r.LogCacheStats {
+		go r.logCacheStatistics()
+	}
+	r.Log.Debugf("cache timeout: seconds=%d\n", int(time.Duration(r.CacheTTL).Seconds()))
```

Comment:
two requests:

1. Can we only print the cache is actually used (e.g. ttl > 0)
2. Can you add the size to the debug output?

It is helpful when getting logs and possibly not a config during triage/issues

Reply:
@powersj for the second item, did you want something other than what is emitted by the logCacheStatistics function?

Reply:
No, that size works. The reason I wanted it added here along with the TTL time is since this happens early on.

Reply:
oh! the cache size! i'm dumb. excuse the brain fog.